zipkin-lens/src/components/TracePage/Header/Header.tsx

```diff
@@ -47,7 +47,8 @@ const useStyles = makeStyles((theme) => ({
   },
   infoCell: {
     display: 'flex',
-    fontWeight: theme.typography.fontWeightRegular,
```

Comment:
A type error is occurring for some reason.
I think this will probably be resolved by updating the Material UI version.